### ISSUE TITLE
Make synced collections less "obscure" about sync condition

### ISSUE THREAD

**Issue by RafaPolit** [huridocs/uwazi#4301]
We need to make synced collections a little less "obscured" to users:
- notify on template editing that template is synced and changes may brake the sync
- etc

Discuss which steps to take.

Related to #4286

**Comment by LaszloKecskes** [comment]
@Rafa Sync currently tags synced templates, and in the UI you can only set the pages for those entities, but not edit them in any other way. Do you think this is still relevant or we can close this?

**Comment by txau** [comment]
@RafaPolit @LaszloKecskes now this is interesting because I remember we added locking mechanisms for synced templates. @LaszloKecskes did you confirm this feature is still working as of today?

@RafaPolit I was wondering, if it is possible to do bidirentional syncing, of sorts. A parent instance has the template definitions (including thesauri and rel types) and syncs towards the children instances. On the other hand, children have entities that synced towards the parent instance.

This way, templates are locked and controlled by a parent instance, while children can create entities that are going to use the same IDs etc.

**Comment by LaszloKecskes** [comment]
@txau Yes, we worked on a sync issue just this week and it was working fine. Additionally, this is the PR that introduced the changes:
https://github.com/huridocs/uwazi/pull/4487
This comes two months later than, so it's feasible that we just didn't tie it to this issue. I was only asking in case @rafa found something else as well, but if this issue is only about preventing edits to synced templates, then we should close it.

**Comment by aphilop** [comment]
This is resolved
